cognite_toolkit/_cdf_tk/feature_flags.py

```diff
@@ -51,6 +51,10 @@ class Flags(Enum):
         "visible": True,
         "description": "Enables the support for purging datasets and spaces",
     }
+    BUILD_OFFLINE: ClassVar[dict[str, Any]] = {
+        "visible": True,
+        "description": "Enables the support for enforcing building offline.",
```

Comment:
I think this needs a better explaination: what do we mean by "offline" and what's the purpose